examples/ENS/ENS.xml

```diff
@@ -44,116 +48,99 @@
   </ts:origins>
 
   <ts:cards>
-    <ts:action>
-      <ts:name>
-        <ts:string xml:lang="en">Refund</ts:string>
-      </ts:name>
-      <ts:transaction>
-        <ts:ethereum function="releaseDeed" contract="ENS">
-          <ts:data>
-            <ts:bytes32 ref="tokenId"/>
-          </ts:data>
-        </ts:ethereum>
-      </ts:transaction>
-      <ts:view xml:lang="en">
-        <xhtml:style type="text/css">&style;</xhtml:style>
-        <xhtml:script type="text/javascript">&release.en;</xhtml:script>
-      </ts:view>
-    </ts:action>
 
     <ts:action>
       <ts:name>
-        <ts:string xml:lang="en">Renew</ts:string>
+        <ts:string xml:lang="en">Records</ts:string>
       </ts:name>
 
-      <ts:attribute-type id="renewalPricePerYear" syntax="1.3.6.1.4.1.1466.115.121.1.36">
+      <ts:attribute-type id="value" syntax="1.3.6.1.4.1.1466.115.121.1.36">
```

Comment:
@James-Sangalli  Is this meant to be the new email address? Then it should be `1.3.6.1.4.1.1466.115.121.1.15` instead of `1.3.6.1.4.1.1466.115.121.1.36`